src/main/resources/application-local.yml

```diff
@@ -37,17 +37,23 @@ person-record-service:
 hmpps.sqs:
   enabled: true
   provider: localstack
+  region: eu-west-2
   queues:
     courtcasesqueue:
-      queueName: court-cases-queue.fifo
+      queueUrl: http://localhost:4566/000000000000/court_cases_queue.fifo
+      queueName: court_cases_queue.fifo
       subscribeTopicId: courtcasestopic
   topics:
     courtcasestopic:
-      arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}.fifo
+      arn: "arn:aws:sns:eu-west-2:000000000000:court-cases-topic.fifo"
 
 feature:
   flags:
-    pass-hearing-id-to-court-case-service: true
```

Comment:
👍 although I would be inclined to do this in a separate PR

Reply:
https://github.com/ministryofjustice/court-case-matcher/pull/705